wasm/jit/jit_amd64_test.go

```diff
@@ -735,6 +747,49 @@ func TestAmd64Builder_handlePick(t *testing.T) {
 	})
 }
 
+func TestAmd64Builder_handleConstI32(t *testing.T) {
+	for _, v := range []uint32{1, 1 << 5, 1 << 31} {
+		t.Run(fmt.Sprintf("%d", v), func(t *testing.T) {
+			o := &wazeroir.OperationConstI32{Value: v}
+			builder := requireNewBuilder(t)
+			builder.initializeReservedRegisters()
+			// Dummy value not used!
```

Comment:
do we mean this will be overwritten with the constant later? what happens if we don't execute this pushValueOnStack (may lead to a better comment)

Reply:
deleted the dummies as they are not important :D